Content.Tests/DMProject/Tests/Const/Const_LateDefinition.dm

```diff
@@ -0,0 +1,13 @@
+var/global/list/global_list = list(late_defined_const)
+
+var/const/late_defined_const3 = late_defined_const2
+var/const/late_defined_const2 = late_defined_const
+var/const/late_defined_const = 1
+
+/obj/TestObj
+	var/a = 1 + const_var
+	var/const/const_var = 2
+
+/proc/RunTest()
+	var/obj/TestObj/o = new
+	return o.const_var + global_list[1]
```

Comment:
A test's return value isn't used for anything if there isn't a `// RETURN TRUE` comment at the top of the file. Instead, `ASSERT()` that the values are what they should be.
```suggestion
	ASSERT(o.const_var == 2)
	ASSERT(global_list[1] == 1)
```

Reply:
Done